docs/source/models.rst

```diff
@@ -22,819 +22,27 @@ keypoint detection, video classification, and optical flow.
-    :template: function.rst
-
-    torchvision.models.detection.retinanet_resnet50_fpn
-
-
-SSD
----
-
-.. autosummary::
-    :toctree: generated/
-    :template: function.rst
-
-    torchvision.models.detection.ssd300_vgg16
-
-
-SSDlite
--------
-
-.. autosummary::
-    :toctree: generated/
-    :template: function.rst
-
-    torchvision.models.detection.ssdlite320_mobilenet_v3_large
-
-
-Mask R-CNN
-----------
-
-.. autosummary::
-    :toctree: generated/
-    :template: function.rst
-
-    torchvision.models.detection.maskrcnn_resnet50_fpn
-
-
-Keypoint R-CNN
---------------
-
-.. autosummary::
-    :toctree: generated/
-    :template: function.rst
-
-    torchvision.models.detection.keypointrcnn_resnet50_fpn
-
-
-Video classification
-====================
-
-We provide models for action recognition pre-trained on Kinetics-400.
-They have all been trained with the scripts provided in ``references/video_classification``.
-
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB videos of shape (3 x T x H x W),
-where H and W are expected to be 112, and T is a number of video frames in a clip.
-The images have to be loaded in to a range of [0, 1] and then normalized
-using ``mean = [0.43216, 0.394666, 0.37645]`` and ``std = [0.22803, 0.22145, 0.216989]``.
-
-
-.. note::
-  The normalization parameters are different from the image classification ones, and correspond
-  to the mean and std from Kinetics-400.
-
-.. note::
-  For now, normalization code can be found in ``references/video_classification/transforms.py``,
-  see the ``Normalize`` function there. Note that it differs from standard normalization for
-  images because it assumes the video is 4d.
-
-Kinetics 1-crop accuracies for clip length 16 (16x112x112)
-
-================================  =============   =============
-Network                           Clip acc@1      Clip acc@5
-================================  =============   =============
-ResNet 3D 18                      52.75           75.45
-ResNet MC 18                      53.90           76.29
-ResNet (2+1)D                     57.50           78.81
-================================  =============   =============
-
-
-ResNet 3D
-----------
-
-.. autosummary::
-    :toctree: generated/
-    :template: function.rst
-
-    torchvision.models.video.r3d_18
-
-ResNet Mixed Convolution
-------------------------
-
-.. autosummary::
-    :toctree: generated/
-    :template: function.rst
-
-    torchvision.models.video.mc3_18
-
-ResNet (2+1)D
--------------
-
-.. autosummary::
-    :toctree: generated/
-    :template: function.rst
-
-    torchvision.models.video.r2plus1d_18
-
-Optical flow
-============
-
-Raft
-----
-
-.. autosummary::
-    :toctree: generated/
-    :template: function.rst
-
-    torchvision.models.optical_flow.raft_large
-    torchvision.models.optical_flow.raft_small
+TODO: Something similar to classification models: list of models + table of weights
```

Comment:
We need to consider Quantized Classification models like a separate submodule (similar to detection and classification) and add documentations, tables with accuracies etc.